### PR TITLE
HG-51: Text Link component

### PR DESCRIPTION
**Pull request by AlexP-anec**

### CODE REVIEW

File: src/components/TextLink/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@further-ventures/fv-react-ui-components.textlink",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "classnames": ">=2.0.0",
+    "react": ">=17.0.0",
+    "react-jss": ">=10.0.0",
+    "@further-ventures/fv-react-ui-components.theme": ">=0.0.1"
```
Comment (SergeySychevskiy) [inline src/components/TextLink/package.json:7]: I think you don't need react-jss as well as theme as a dependency

File: export.config.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "Input": "input",
     "Form": "form",
     "Icons": "icons",
-    "Core": "core"
+    "Core": "core",
+    "TextLink": "textlink"
```
Comment (Nagibaba) [inline export.config.json]: the last element of a json should contain a `,` for git diff

File: src/components/TextLink/index.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  return <>
+    {
+      disabled ? 
+        <span
+          className={classes}
+        >
+          {textLinkContent}
+        </span>
+        :
+        typeOfUrl(to) === 'internal' ?
```
Comment (Nagibaba) [inline src/components/TextLink/index.tsx]: so many inner conditions. should be divided into components

File: src/components/TextLink/index.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
```
Comment (dtomaszewski88) [inline src/components/TextLink/index.tsx]: I don't think this component should handle any actual routing logic. This should be just a visual element - whatever routing library the user wants to use, its up to them.

File: src/components/TextLink/index.tsx (hunk 3 above)
Comment (dtomaszewski88) [reply src/components/TextLink/index.tsx]: Not necessary split into components IMO, but at least guard clauses for improved readability. Nested ternaries are something we should avoid

File: export.config.json (hunk 2 above)
Comment (dtomaszewski88) [reply export.config.json]: I agree, but its not feasible to add it everytime manually. I'll be making improvements to our husky/eslint setup to take care of that

File: src/components/TextLink/index.tsx (hunk 4 above)
Comment (AlexP-anec) [reply src/components/TextLink/index.tsx]: ok :) so just plain anchor

File: src/components/TextLink/package.json [hunk 5]
```diff
@@ -4,7 +4,6 @@
   "peerDependencies": {
     "classnames": ">=2.0.0",
     "react": ">=17.0.0",
-    "react-jss": ">=10.0.0",
     "@further-ventures/fv-react-ui-components.theme": ">=0.0.1"
```
Comment (dtomaszewski88) [inline src/components/TextLink/package.json:7]: Theme is also not used here anymore, right?

**Review by dtomaszewski88 — APPROVED** [review]

**Pull request merged**